integration-test/src/test/java/org/terracotta/angela/VoterTest.java

```diff
@@ -51,20 +47,25 @@ public void testVoterStartup() throws Exception {
                                 .configRepo("terracotta1/repository")
                                 .logs("terracotta1/logs")
                                 .metaData("terracotta1/metadata")
+                                .failoverPriority("consistency:1"),
+                            server("server-2", "localhost")
+                                .tsaPort(9510)
+                                .tsaGroupPort(9511)
+                                .configRepo("terracotta2/repository")
+                                .logs("terracotta2/logs")
+                                .metaData("terracotta2/metadata")
                                 .failoverPriority("consistency:1")
                         )
                     )
                 )
             )
-        ).voter(
-            voter -> voter.distribution(distribution(version("3.9-SNAPSHOT"), PackageType.KIT, TERRACOTTA_OS))
-                          .addVoter(TerracottaVoter.voter("voter", "localhost", "localhost:9410"))
+        ).voter(voter -> voter
+            .distribution(distribution(version("3.9-SNAPSHOT"), KIT, TERRACOTTA_OS))
+            .addVoter(voter("voter", "localhost", "localhost:9410"))
         );
 
-    try (ClusterFactory factory = new ClusterFactory("ConfigToolTest::testVoterStartup", configContext)) {
-      Tsa tsa = factory.tsa();
-      tsa.startAll().activateAll();
-
+    try (ClusterFactory factory = new ClusterFactory("VoterTest::testVoterStartup", configContext)) {
+      factory.tsa().startAll().activateAll();
       factory.voter().startAll();
```

Comment:
lets ensure voter moves to `CONNECTED_TO_ACTIVE` state

Reply:
Done